client/components/button/style.scss

```diff
@@ -209,6 +209,17 @@ button {
 			height: 18px;
 			top: 5px;
 		}
+
+		//arrows sit 1px low in their svg bounding box, need a nudge
```

Comment:
I'm not sure, it just looks very low, especially compared to other icons. I would guess it's because of the horizontal line needing to be either above or below the center of the 24px box.